### PR TITLE
feat: migrate facebook env vars to globalConfig

### PR DESCRIPTION
**Pull request by vishnu-narayanan**
# Pull Request Template

## Description

Migrate env vars to globalConfig and make it editable from the super admin UI. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## How Has This Been Tested?

- Tested on staging chatwoot env. FB credentials gets listed on GlobalConfig and the FB functionality is working fine,

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/controllers/super_admin/installation_configs_controller.rb [hunk 1]
```diff
@@ -2,10 +2,15 @@ class SuperAdmin::InstallationConfigsController < SuperAdmin::ApplicationControl
   # Overwrite any of the RESTful controller actions to implement custom behavior
   # For example, you may want to send an email after a foo is updated.
   #
-  # def update
-  #   super
-  #   send_foo_updated_email(requested_resource)
-  # end
+  def update
```
Comment (sojan-official) [inline app/controllers/super_admin/installation_configs_controller.rb:5]: lets move this to a `after_commit` callback in the installation config model instead

**Comment by vishnu-narayanan** [comment]
Tested the facebook part on staging. The keys are getting listed in GlobalConfig and functionality is working fine. 
image

**Comment by vishnu-narayanan** [comment]
@sojan-official Can we merge this and have the UI change to group the FB config together as an iterative PR ?

**Comment by vishnu-narayanan** [comment]
@tejaswinichile done

File: app/controllers/webhooks/instagram_controller.rb [hunk 2]
```diff
@@ -25,6 +25,6 @@ def events
   private
 
   def valid_instagram_token?(token)
-    token == ENV['IG_VERIFY_TOKEN']
+    token == GlobalConfigService.load('FB_VERIFY_TOKEN', '')
```
Comment (tejaswinichile) [inline app/controllers/webhooks/instagram_controller.rb:28]: No No it should be `GlobalConfigService.load('IG_VERIFY_TOKEN', '')`

Comment (vishnu-narayanan) [reply app/controllers/webhooks/instagram_controller.rb]: My bad. Fixed.

**Comment by sojan-official** [comment]
@vishnu-narayanan  lets revert the view files as we are not changing them

**Comment by vishnu-narayanan** [comment]
@sojan-official done

**Pull request merged**